docs/integration.rst

```diff
@@ -400,7 +400,7 @@ provides a dedicated boot source selection framework, called *bootchooser*,
 backed by an atomic and redundant storage backend, named *state*.
 
 *Barebox state* allows you to save the variables required by bootchooser with
-memory specific storage strategies in all common storage medias,
+memory specific storage strategies in all common storage mediums,
```

Comment:
Shouldn't that be media?

Reply:
[Both are correct](https://en.wiktionary.org/wiki/medium#Noun).